### ISSUE TITLE
Condense the fallback case and regular case in `fallback`

### ISSUE THREAD

**Issue by JakeNavith** [JakeNavith/tailwindcss-theme-variants#1]
For example:

```js
{
  themes: {
    light: { mediaQuery: prefersLight },
    dark: { mediaQuery: prefersDark },
  },
  fallback: true
}
```

Generates this:
```css
bg-whatever {
  background-color: #whatever;
}

.light\:bg-whatever {
  background-color: #whatever;
}

/* This one isn't needed */
@media (prefers-color-scheme: light) {
  .light\:bg-whatever {
    background-color: #whatever;
  }
}

@media (prefers-color-scheme: dark) {
  .dark\:bg-whatever {
    background-color: #whatever;
  }
}
```

We can reclaim unnecessarily lost bytes by removing that unneeded case.

**Comment by JakeNavith** [comment]
For 2.0, make this the default since CSS file size is a big concern with theming this complex.

**Comment by JakeNavith** [comment]
Closed by 050b618694fd2b1523d7db33c60f658e61ce5176, 1d3f271eebc5b0d44b3f238c50d7221bc12962b9, and fe24b9d3cf4e679b8c642b3b739b8889def34b6c
